index.html

```diff
@@ -0,0 +1,1924 @@
+                  ><img
+                    class="round-opinion"
+                    src="images/opinion4.png"
+                    alt="opinion1"
+                /></a>
+                <div class="opinion-info">
+                  <h5>
+                    On Memorial Day, Remember Why Americans Made the Ultimate
+                    Sacrifice
+                  </h5>
+                  <h4>BY NEWT GINGRICH</h4>
+                </div>
+              </div>
+              <div class="opinion-sec d-flex">
+                <a href="#"
+                  ><img
+                    class="round-opinion"
+                    src="images/opinion5.png"
+                    alt="opinion1"
+                /></a>
+                <div class="opinion-info">
+                  <h5>
+                    On Memorial Day, Remember Why Americans Made the Ultimate
+                    Sacrifice
+                  </h5>
+                  <h4>BY NEWT GINGRICH</h4>
+                </div>
+              </div>
+              <div class="opinion-sec d-flex">
+                <a href="#"
+                  ><img
+                    class="round-opinion"
+                    src="images/opinion6.png"
+                    alt="opinion1"
+                /></a>
+                <div class="opinion-info">
+                  <h5>
+                    On Memorial Day, Remember Why Americans Made the Ultimate
+                    Sacrifice
+                  </h5>
+                  <h4>BY NEWT GINGRICH</h4>
+                </div>
+              </div>
+            </div>
+          </div>
+
+          <div class="spons2 container-fluid no-padding">
+            <!-- SPONSORED 2 -->
+            <article class="card card2">
+              <div class="img">
+                <a href="#"
+                  ><img
+                    class="card-img-bottom"
+                    src="images/spons11.jpg"
+                    alt="Card image"
+                    style="width: 100%;"
+                /></a>
+                <div class="cat cat2">SPONSORED INSIGHT</div>
+              </div>
+              <div class="card-body pad">
+                <h3 class="card-title">
+                  Looking for a Powerful, Lightweight and Long-lasting Cordless
+                  Vacuum?
+                </h3>
+                <p class="card-text">
+                  Roborock H6, 2020 Red Dot Award Winner, Ultra-Long lasting
+                  LiPo battery, available now on Amazon.
+                </p>
+              </div>
+            </article>
+            <form
+              class="subscribe form-inline d-flex flex-column justify-content-center"
+            >
+              <div
+                class="form-group d-flex align-items-center justify-content-between"
+              >
+                <img class="flex-shrink-0" src="images/news-icon.png" alt="" />
+                <h3 class="darkh3">Start your day with our top 5 articles</h3>
+              </div>
```

Comment:
The image and the heading should be beside each other. Currently, they are misaligned.
![Annotation 2020-06-04 121048](https://user-images.githubusercontent.com/57726348/83723649-a42f8080-a65c-11ea-9c03-a3d250630070.jpg)